src/build.py

```diff
@@ -731,9 +731,13 @@ def CMakeCommandBase():
   command.append('-DPYTHON_EXECUTABLE=%s' % sys.executable)
   command.append('-DCMAKE_EXPORT_COMPILE_COMMANDS=ON')
   command.append('-DCMAKE_BUILD_TYPE=Release')
-  if IsMac:
+  if IsMac():
     # Target macOS Seirra (10.12)
     command.append('-DCMAKE_OSX_DEPLOYMENT_TARGET=10.12')
+  elif IsWindows():
+    # CMake's usual logic fails to find LUCI's git on Windows
+    git_exe = proc.check_output(['where', 'git']).strip()
+    command.append('-DGIT_EXECUTABLE=%s' % git_exe)
```

Comment:
Oh come on.  Really camke?   How can you not do this.

Also, do we not already a have a find_in_path type function in this project?

Reply:
Wabt uses FIND_PACKAGE (https://github.com/WebAssembly/wabt/blob/master/CMakeLists.txt#L70) but it doesn't seem to work. To be fair, LUCI's setup is weird, especially on Windows.

Reply:
Oh you're right though, we have `proc.which`. I'll try that.